web/src/utils/transformDataHelper.tsx

```diff
@@ -64,10 +64,40 @@ export function transformForm(data: any) {
   }
 };
 
-export function transformProfile(profileData: any): any[] | [] {
+// sort the list of profiles from the latest to the earliest update_at
+export function sortProfileByDatetime(profileData: any): any[] | [] {
   try {
     return profileData.sort((a: any, b: any) => { return Date.parse(b.updatedAt) - Date.parse(a.updatedAt) });
```

Comment:
This might work `return profileData.sort((a: any, b: any) => Date.parse(b.updatedAt) - Date.parse(a.updatedAt));`